iOS Example Frame/iOS Example Frame/Style/OtherStyle.swift

```diff
@@ -0,0 +1,274 @@
+//
+//  OtherStyle.swift
+//  iOS Example Frame
+//
+//  Created by Alex Ioja-Yang on 18/08/2022.
+//  Copyright © 2022 Checkout. All rights reserved.
+//
+import UIKit
+import Frames
+
+enum OtherStyle {
+
+    // MARK: Matrix Style with one function
+    static func buildWithOneFunction() -> FramesStyle {
```

Comment:
I cannot agree with this PR as discussed previously. Happy to be overridden if need be, but previous ThemeDemo was 76 lines all in, with nearly 60 lines of linear code describing the components in sequence and "demoing the Theme". 

You're proposing replacing it with nearly 300 lines of complex unlinear code that negates the reason we created Theme (convenient simple way to declare the UI config).

Additionally, we should not put MARK on top of every function, it doesn't make sense. MARK exists to break down code into groups for easier navigation. If you break down into nearly every function, then the function itself was already the "mark".